README.md

```diff
@@ -249,10 +283,10 @@ Creates a Button in the menu to a submenu
 #### Usage
 
 As methods return the current menu you can concat button methods like that:
+
 ```js
 const menu = new TelegrafInlineMenu('test')
-menu.manual('Test 1', 'a')
-  .manual('Test 2', 'b')
+menu.manual('Test 1', 'a').manual('Test 2', 'b')
```

Comment:
I prefer multiple Buttons in multiple Rows. This example is simple but as examples get copied faster than I often expect they will result in confusing code for the just started user.

```suggestion
menu
  .manual('Test 1', 'a')
  .manual('Test 2', 'b')
```